lib/src/offline/route_mapping.dart

```diff
@@ -0,0 +1,80 @@
+final routes = [
```

Comment:
I was thinking this could be generated by the SDK generator.

Reply:
Well, i'm okay with generating by SDK generator if that is easier than hardcoding.